Makefile

```diff
@@ -61,26 +61,37 @@ bin/brogue.exe: $(objects) windows/icon.o
 	mt -manifest windows/brogue.exe.manifest '-outputresource:bin/brogue.exe;1'
 
 clean:
-	$(RM) src/brogue/*.o src/platform/*.o bin/brogue{,.exe}
+	$(RM) src/brogue/*.o src/platform/*.o windows/icon.o bin/brogue{,.exe}
```

Comment:
Will `rm` fail here if `windows/icon.o` is not present?

Reply:
No, `$(RM)` is `rm -f` so ignores non-existent arguments. That's why the `{,.exe}` one works too